app/scicrunch_process_results.py

```diff
@@ -21,6 +21,12 @@ def _prepare_results(results):
         attr = _transform_attributes(attributes_map, hit)
         attr['doi'] = _convert_doi_to_url(attr['doi'])
         attr['took'] = results['took']
+        # find context files by looking through object mimetypes
+        attr['abi-contextual-information'] = [
+            file['dataset']['path']
+            for file in hit['_source']['objects']
+            if file['additional_mimetype']['name'].find('context') is not -1
```

Comment:
Why only look for the partial mimetype here? Why not 'x.vnd.abi.context-information'?

Reply:
I'm only looking through the partial mimetype since I know it is a bout to change

Reply:
I would like to see this using the largest possible common matching term.  In this case it would be `abi.context-information`.